rama-net/src/tls/mod.rs

```diff
@@ -95,3 +95,58 @@ pub enum DataEncoding {
     /// Privacy Enhanced Mail (PEM) (plain text)
     Pem(NonEmptyString),
 }
+
+#[cfg(feature = "boring")]
```

Comment:
You should be able to implement these directly in boring/rustls modules. Child modules are AFAIK not subject to the orphan rule, but might recall this wrongly so.

Reply:
Implemented this in the root module as there are currently no separate modules for boring and rustls at this level. Would need a decent refactor. But as a rework is planned in the future for tls this could be done then